src/libsync/vfs/cfapi/vfs_cfapi.cpp

```diff
@@ -358,8 +358,8 @@ void VfsCfApi::requestHydration(const QString &requestId, const QString &path)
 
     // Set in the database that we should download the file
     SyncJournalFileRecord record;
-    journal->getFileRecord(relativePath, &record);
-    if (!record.isValid()) {
+    const auto result = journal->getFileRecord(relativePath, &record);
+    if (!result || !record.isValid()) {
```

Comment:
@mgallien Do you need to store that `result`? If not, maybe you could just go:
`if (!journal->getFileRecord(relativePath, &record) || !record.isValid())`?

Reply:
done